xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.2003-122.xml

```diff
@@ -151,10 +155,10 @@
         <body>
             <div type="edition">
             <ab>
-                <lb n="1"/><supplied reason="lost"><g type="cross"/><!-- check this -->ἐ</supplied><unclear>πὶ</unclear> <persName><name key="Ephraemius Ἐφραιμίος"><!-- check name -->Ἐφραιμ<supplied reason="lost">ίου</supplied></name></persName><!-- extra? --><gap reason="lost" extent="unknown" unit="character"/>
-                <lb n="2"/><supplied reason="lost">Θ</supplied>είων λ<supplied reason="lost">αργιτιώνων</supplied><!-- extra? --><gap reason="lost" extent="unknown" unit="character"/>
+                <lb n="1"/><supplied reason="lost">ἐ</supplied><unclear>πὶ</unclear> <persName><name key="Ephraemius Ἐφραιμίος"><!-- what name is this? -->Ἐφραιμ<supplied reason="lost">ίου</supplied></name></persName><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
It's the name Ephraim, a name drawn from the Bible; since it's not originally Greek, that might explain some of the oddity of it. Your mark-up is good though!